### DISCUSSION TITLE
Open MCT User's Guide

### DISCUSSION THREAD

**Discussion by tiffanyatruong** [nasa/openmct#3769] [Announcements]
The Open MCT User's Guide functions as a helpful tool to direct users as they learn to navigate, use, and troubleshoot Open MCT. Users can rapidly view and explore data in a variety of visualizations, as well as create many different types of views that can be saved and shared with others. This latest release includes updates on how to use and display imagery metadata and activity plans, as well as how to create a Timestrip View. To view or download the User's Guide, click [here](https://nasa.github.io/openmct/static/files/Open_MCT_Users_Guide.pdf).

**Comment by jvigliotta** [comment]
Congrats to @charlesh88 and @tiffanyatruong for all their hard work on putting that together. Awesome job guys!
